assets/scss/_variables_project.scss

```diff
@@ -0,0 +1,11 @@
+$secondary: #303030;
```

Comment:
Should we set `$primary` too?

Reply:
The value of `$primary` has been the same as docsy and so I didn't change it

site
https://github.com/kubernetes/contributor-site/blob/cea0531aa755544a936e69b22098f3231510f940/assets/scss/_variables.scss#L20

docsy
https://github.com/google/docsy/blob/25f73360dc4b72c7606531b37edace0417b40858/assets/scss/_variables.scss#L20

If the primary colour needs to be changed, we can set the `$primary` and maybe others too.

Reply:
We should use `#326CE5` - see https://github.com/kubernetes/kubernetes/blob/master/logo/colors.md

If we don't use that already, oops, let's get a small PR merged so that we do, and then rebase / adapt this one.
I checked and we seem to use `#3176D9`. Maybe that's what early Docsy defaults to?

Reply:
Sorry, I can't figure out where we use `#3176D9`. I checked and it cannot be `$primary-light` as it lightens `$primary` by 75% which is practically white when `$primary` is `#3063de`.

Reply:
Well, I would expect `$primary` to be `#326ce5`; can we do the small fix first and rebase this big one?

Reply:
Sure, I will create a new PR setting the new primary.

Reply:
Apologies for the delay but I have created the PR for the colour at #555

Reply:
This is resolved, rebased and pushed.